dspace-api/src/main/java/org/dspace/app/util/AuthorizeUtil.java

```diff
@@ -525,4 +532,73 @@ public static void authorizeReinstateItem(Context context, Item item)
             }
         }
     }
+
+    /**
+     * This method will check whether the current user is authorized to manage the default read group
+     * @param context       The relevant DSpace context
+     * @param collection    The collection for which this will be checked
+     * @throws AuthorizeException   If something goes wrong
+     * @throws SQLException If something goes wrong
+     */
+    public static void authorizeManageDefaultReadGroup(Context context,
+                                                      Collection collection) throws AuthorizeException, SQLException {
+        AuthorizeService authorizeService = AuthorizeServiceFactory.getInstance().getAuthorizeService();
+        authorizeService.authorizeAction(context, collection, Constants.ADMIN);
+    }
+
+    /**
+     * This method checks whether the current user has sufficient rights to modify the group.
+     * Depending on the kind of group and due to delegated administration, separate checks need to be done to verify
+     * whether the user is allowed to modify the group.
+     *
+     * @param context the context of which the user will be checked
+     * @param group   the group to be checked
+     * @throws SQLException
+     * @throws AuthorizeException
+     */
+    public static void checkAuthorizationOnGroup(Context context, Group group) throws SQLException, AuthorizeException {
```

Comment:
Could we name this method similar to the other methods in this Util class?  It probably should be named something like `authorizeManageGroup`.... as you can see almost all utility methods in this class start with `authorizeManage*` followed by the thing they are checking management permissions on.